src/components/admin/adminInquiry/AdminInquiryAnswerWrite.tsx

```
@@ -45,6 +45,13 @@ export default function AdminInquiryAnswerWrite() {
     }
   }, [answerData]);
 
+  useEffect(() => {
+    if (inputRef && inputRef.current) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)